sway/desktop/output.c

```diff
@@ -199,6 +199,14 @@ void output_layer_for_each_toplevel_surface(struct sway_output *output,
 	wl_list_for_each(layer_surface, layer_surfaces, link) {
 		struct wlr_layer_surface_v1 *wlr_layer_surface_v1 =
 			layer_surface->layer_surface;
+		struct fx_render_data *data = user_data;
+		struct layer_effects *effects = layer_surface->effects;
+		if (effects) {
```

Comment:
Would the effects struct always be defined? Could remove this check

Reply:
Nah, it's a pointer and only gets defined when a suitable `layer_effects` config is found